Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1417,6 +1417,9 @@ - (BOOL)_scrollToRect:(WebCore::FloatRect)targetRect origin:(WebCore::FloatPoint
     WebCore::FloatPoint unobscuredContentOffset = unobscuredContentRect.location();
     WebCore::FloatSize contentSize([self._currentContentView bounds].size);
 
+    auto scrollViewInsets = [_scrollView adjustedContentInset];
```

Comment:
Nit - we use `-_initialContentOffsetForScrollView` for similar use cases — I wonder if that works here as well?

Reply:
I don't think that would work for a couple reasons:
- `_initialContentOffsetForScrollView` uses `_computedContentInset`
-  `_computedContentInset` simply returns `_obscuredInsets` if they are set, ignoring the additional system inset due to the find bar
- `-[WKWebView _scrollViewSystemContentInset]` also does not account for the additional system inset (this may have other implications on its own)
- `contentOffsetBoundedInValidRange`, which has a similar use case, uses `[_scrollView adjustedContentInset]` for the minimum

Do you think there are any risks to using the total inset?

Reply:
Ah, okay. In that case, let's stick with `-adjustedContentInset`.

> …ignoring the additional system inset due to the find bar

Hm... I do wonder if this means we should be using that in 
`-_initialContentOffsetForScrollView `, so that the initial scroll position after top navigation will respect insets due to the find bar being present.

Reply:
Not a specific concern for find, since we dismiss the bar on navigation, but I've left a `FIXME` behind, since it does seem generally incorrect.